notebooks/002-openvino-api/002-openvino-api.ipynb

```diff
@@ -461,7 +461,7 @@
    "id": "53bf7c61",
    "metadata": {},
    "source": [
-    "`.infer()` sets output tensor, that we can reach using `get_output_tensor()`. Since we know this network returns one output, and we stored the reference to the output layer in the `output_layer.index` parameter, we can get the data with `request.get_output_tensor(output_layer.index)`. To get numpy array from output we need to take parameter `.data`."
+    "The `.infer()` sets output tensor, that can be reached, using the `get_output_tensor()`. Since this network returns one output, and the reference to the output layer is in the `output_layer.index` parameter, you can get the data with `request.get_output_tensor(output_layer.index)`. To get a numpy array from the output, use the `.data` parameter."
```

Comment:
```suggestion
    "The `.infer()` function sets output tensor, that can be reached, using `get_output_tensor()`. Since this network returns one output, and the reference to the output layer is in the `output_layer.index` parameter, you can get the data with `request.get_output_tensor(output_layer.index)`. To get a numpy array from the output, use the `.data` parameter."
```